src/lib/dates.test.jsx

```diff
@@ -0,0 +1,40 @@
+import { describe, it, expect, vi } from "vitest";
+import { getDayOfWeek } from "./dates";
+
+describe("getDayOfWeek", () => {
+  it("should return 'Tomorrow' for the date after today", () => {
+    const mockDate = new Date(2024, 0, 7);
+    vi.setSystemTime(mockDate);
+
+    const dayIndex = (mockDate.getDay() + 1) % 7;
+    expect(getDayOfWeek(dayIndex)).toBe("Tomorrow");
+  });
+
+  it("should return the correct day of the week", () => {
+    const dayIndex = 2;
+    expect(getDayOfWeek(dayIndex)).toBe("Tuesday");
+  });
+
+  it('should return "Sunday" for index 0', () => {
+    expect(getDayOfWeek(0)).toBe("Sunday");
+  });
+
+  it('should not return "Tomorrow" for any index other than the next day', () => {
+    const today = new Date().getDay();
+    for (let i = 0; i < 7; i++) {
+      if (i !== (today + 1) % 7) {
+        expect(getDayOfWeek(i)).toBe(
+          [
+            "Sunday",
+            "Monday",
+            "Tuesday",
+            "Wednesday",
+            "Thursday",
+            "Friday",
+            "Saturday",
+          ][i]
+        );
+      }
+    }
+  });
```

Comment:
Same situation here, what I'd is:

Update your `dates` method to have the days separated as a constant variable:

```js
export const DAYS_OF_WEEK = [
  "Sunday",
  "Monday",
  "Tuesday",
  "Wednesday",
  "Thursday",
  "Friday",
  "Saturday",
];

/**
 * Returns the day of the week or "Tomorrow" if the day is the next day.
 * @param {number} dayIndex - Index of the day.
 * @returns {string} - Day of the week or "Tomorrow".
 */
export const getDayOfWeek = (dayIndex) => {
  const today = new Date().getDay();
  const tomorrow = (today + 1) % 7;
  return dayIndex === tomorrow ? "Tomorrow" : DAYS_OF_WEEK[dayIndex];
};
```

And then inside your test:

```js
it('should not return "Tomorrow" for any index other than the next day', () => {
  const today = new Date().getDay();
  const tomorrowIndex = today + 1;

  DAYS_OF_WEEK.forEach((day, index) => {
    if (index !== tomorrowIndex) {
      expect(getDayOfWeek(index)).toBe(day);
    }
  });
});
 ```